frontend/src/metabase/admin/upsells/UpsellBetterSupport.tsx

```diff
@@ -0,0 +1,31 @@
+import { c, t } from "ttag";
+
+import { getPlan } from "metabase/common/utils/plan";
+import { useSelector } from "metabase/lib/redux";
+import { getSetting } from "metabase/selectors/settings";
+
+import { UpsellCard } from "./components";
+
+export const UpsellBetterSupport = ({ source }: { source: string }) => {
+  const plan = useSelector(state =>
+    getPlan(getSetting(state, "token-features")),
+  );
+
+  if (plan !== "oss") {
+    return null;
+  }
+
+  return (
+    <UpsellCard
+      title={t`Get expert help`}
+      campaign="better-hosting"
+      buttonText={t`Try for free`}
+      buttonLink="https://www.metabase.com/upgrade"
```

Comment:
`frontend/src/metabase/nav/components/StoreLink/StoreLink.tsx` has this same constant for this url. might be worth keeping these in sync.